build-support/bin/generate_github_workflows.py

```diff
@@ -159,12 +182,12 @@ def expose_all_pythons() -> Sequence[Step]:
     ]
 
 
-def test_workflow_jobs(python_versions: Sequence[str]) -> Jobs:
+def test_workflow_jobs(primary_python_version: str) -> Jobs:
```

Comment:
Nit: this was a list because we expect to add 3.9 to cron sometime soon.

Reply:
Ack, although I'm thinking we can add that back once it's ready. There are some unknowns still like that `native_engine.so` is different across Py38 and Py39, so we need to cache it twice. It's not as simple as setting `python_versions` to both Py38 and Py39 to run them both in cron.

For now, this makes the Build Wheels code a bit more clear because there, we will always activate 3.7, 3.8, and 3.9 at the same time.

Reply:
> Ack, although I'm thinking we can add that back once it's ready. There are some unknowns still like that native_engine.so is different across Py38 and Py39, so we need to cache it twice. It's not as simple as setting python_versions to both Py38 and Py39 to run them both in cron.

It's a matrix job, so it _might_ be that simple...? 🤞